src/config/ssd.md

```diff
@@ -0,0 +1,147 @@
+# Solid State Drives
+
+Post installation, you will need to enable TRIM for solid state drives. You can
+check which devices allow TRIM by running:
+
+```
+$ lsblk --discard
+```
+
+If the DISC-GRAN (discard granularity) and DISC-MAX (discard maximum bytes)
+columns are non-zero, that means the block device has TRIM support. If your
+solid state drive partition does not show TRIM support, please verify that you
+chose a file system with TRIM support (ext4, btrfs, etc.).
+
+To run TRIM one-shot, you can run
+[`fstrim(8)`](https://man.voidlinux.org/fstrim.8) manually. Note that to run
+`fstrim` on F2FS file systems, you need to be running kernel 4.19 or greater.
+For example, if your / directory is on an SSD:
```

Comment:
I'd prefer if you mentioned F2FS in the previous paragraph.

"(ext4, Btrfs, F2FS, etc.). Note that F2FS requires kernel 4.19 or above to support TRIM."